### PR TITLE
Add Feature: Lora/Embed Syntax Highlighting

### PR DESCRIPTION
**Pull request by ElliottLester**
We now highlight lora and embedding in prompts if they will be accepted by the server.
Green -> Matches exactly one lora/embedding in the server list
Yellow -> Matches more than one lora/embedding on the server and probably won't work
Red -> Doesn't match any lora/embedding on the server

Notes:
- Move the lora and embed regular expression to utils
- Create QPromptHighLighter and add it to SDImgPageBase
- Fetch list of embeddings from server
- REFRESH_INTERVAL is now 15 seconds
- Check result in transparency_mask_inserter()

### CODE REVIEW

File: frontends/krita/krita_comfy/widgets/prompt.py [hunk 1]
```diff
@@ -1,5 +1,27 @@
+import re
+from PyQt5.QtGui import QSyntaxHighlighter, QColor, QColorConstants
```
Comment (JasonS09) [inline frontends/krita/krita_comfy/widgets/prompt.py]: This breaks my plugin, can't import `QColorConstants` on my end. Seems PyQt version is lower than required.

**Review by JasonS09 — CHANGES_REQUESTED** [review]

File: frontends/krita/krita_comfy/widgets/prompt.py (hunk 1 above)
Comment (ElliottLester) [reply frontends/krita/krita_comfy/widgets/prompt.py]: Ok let me look at another way to do it

Comment (ElliottLester) [reply frontends/krita/krita_comfy/widgets/prompt.py]: @JasonS09 can you try it now?

**Review by JasonS09 — APPROVED** [review]

**Comment by JasonS09** [comment]
It works now, thanks!

**Pull request merged**